### ISSUE TITLE
Does not modify export command

### ISSUE THREAD

**Issue by jcassee** [ag14774/poetry-monoranger-plugin#28]
When running `poetry export`, this is the output:

```
The lock file does not exist. Locking.
Running command from monorepo root directory
Updating dependencies
Resolving dependencies... (7.9s)

[Errno 2] No such file or directory: 'project_root/subproject/poetry.lock'
```

Does it make sense to also modify the `export` command?

**Comment by ag14774** [comment]
Currently it doesn't support the export command because I didn't personally have a use case for it but it would definitely make sense.

What behavior would you expect from `poetry export`? Export the root lockfile (so all dependencies of the repo) or select just the ones relevant for the current subproject in the repo?

**Comment by jcassee** [comment]
Good question!

My use-case is the combination with nx.dev, and the nx-python plugin. It generates lockfiles, even when using a shared virtual environment.

Selecting only the relevant dependencies would be purest, of course, but I'm already happy with just exporting the whole root lockfile.

**Comment by ag14774** [comment]
In that case, I think `poetry export` at the root of the repo should still work with no changes for exporting the whole root lockfile. I can try to get the exporting of relevant dependencies only in the near future

**Comment by jcassee** [comment]
Indeed, that works, but the nx-python plugin runs it with in the subproject root and expects a lockfile there. (See the output in my first message.)

I do understand this is a pretty particular case. If you have pointers, I might also take a stab at it myself. I did look at the code, but could not understand it with just a casual glance.

**Comment by jcassee** [comment]
I created a very simple PR that implements the first suggestion: #29

The change required was suspiciously simple, the LockCommand just worked.

I'm not sure what kind of tests would be expected.

**Comment by ag14774** [comment]
Thank you for the PR! Indeed there is not many changes required.

In order to test the command, you need to install this version of the plugin as a plugin for poetry (e.g as editable install) and then try `poetry export` in a repo with the plugin enabled.

I tried it and I get an error:
`LockModifier can only be used for `poetry lock`, `poetry install`, and `poetry update` commands.

The solution would be to update `LockModifier` to also accept `ExportCommand` as a viable choice

> I'm not sure what kind of tests would be expected.

I am currently working on improving the testing so I would let this one go without tests for now. I will have a better test suite next week

**Comment by ag14774** [comment]
(But that reminds me I should add instructions on how to setup the plugin for development! I created an issue as a reminder)

**Comment by ag14774** [comment]
Closing this as it has been resolved with #61 . Version 0.4.0 should work if you still need this feature :)
